src/contract-deployment/config.ts

```diff
@@ -87,6 +89,50 @@ export const makeContractDeployConfig = async (
         await AddressManager.setAddress('OVM_Sequencer', sequencerAddress)
         await AddressManager.setAddress('Sequencer', sequencerAddress)
         await contracts.OVM_CanonicalTransactionChain.init()
+
+        // ENQUEUE EVERYTHING FROM ANOTHER CTC
+        // ~~~~ config ~~~~
+        const otherProviderUrl = 'https://goerli.infura.io/v3/3107027ed29545dbb0f02e3b4bf93f23'
+        const otherAddressManagerAddress = '0x4e46C3d1d7453F42E7132e569C1D31925370344C'
+        const startingBlock = 3858399
+
+        // ~~~~ init objs ~~~~
+        const otherProvider = new JsonRpcProvider(otherProviderUrl)
+        const otherSigner = config.deploymentSigner.connect(otherProvider)
+        const otherAddressManager = getContractFactory('Lib_AddressManager').attach(otherAddressManagerAddress).connect(otherSigner)
+
+        // ~~~~ init addrs ~~~~
+        const otherCtcAddress = await otherAddressManager.getAddress('OVM_CanonicalTransactionChain')
+        const otherCtc = getContractFactory('OVM_CanonicalTransactionChain').attach(otherCtcAddress).connect(otherSigner)
+
+        // ~~~~ filter ~~~~
+        const filter = {
+          address: otherCtc.address,
+          topics: [ethers.utils.id(`TransactionEnqueued(address,address,uint256,bytes,uint256,uint256)`)],
```

Comment:
I believe a helper function exists on `otherCtc` for each event, so you can call `otherCtc.TransactionEnqueued()` which will return the same thing as `ethers.utils.id` and is more safe